frontend/dist/assets/_caldavzap/main.js

```diff
@@ -2008,4 +2008,3 @@ function setAgendaFullscreen () {
   }
   isIframeCaldavzapInFullscreen = !isIframeCaldavzapInFullscreen
 }
-
```

Comment:
I'm not sure the linting will like that

Reply:
We can't see on github but there was two empty lines on the end of this file